generators/app/prompts.js

```diff
@@ -289,3 +321,12 @@ function askForUnitaryEntityOperations(context, done) {
         }
     });
 }
+
+function pushComponentsByEntity(context, answers, entity) {
+    context.props.componentsByEntityConfig.push(entity);
```

Comment:
why push entity at first?

Reply:
If you don't do it `context.props.componentsByEntityConfig[entity]` is undefined and you can't use correctly the `mustGenerateComponent` further.